content/2022/en/LCNC-SEC-08-Data-and-Secret-Handling-Failures.md

```diff
@@ -43,6 +47,16 @@ A developer creates an application using a custom API and hard-codes the API key
 Other developers can access the API key directly.
 Moreover, the API key might leak to the app's client code allowing users to gain direct access to the key.
 
+## Example Attack & Misuse Scenarios - Business Users
+
+### Scenario #1
+
+An application is developed, which stores credentials in a database. The database stores passwords in a human readable format, allowing anybody to view them. An employee with access to this database can view the password of every employee within the company, including highly privileged administrative accounts.
+
+### Scenario #2
+
+A new application is built that stores personal data about EU residents.  The developer is unaware of the GDPR rules that allow a user to have their personal data deleted upon request, so never builds a process to support that need. Rather the requests are processed manually and inconsistently, leading to fines, and later a ban from doing business in the EU. 
```

Comment:
I think this part is somewhat too harsh :)
"and later a ban from doing business in the EU" 

I also suggest mentioning uncomfortable questions asked by auditors

Reply:
Thank you! Rewrite: https://github.com/OWASP/www-project-top-10-low-code-no-code-security-risks/pull/56/commits/43f10dbb772c26791b80556696fc2e85690f14cf